app/feature_defaults_unittest.cc

```diff
@@ -236,7 +236,6 @@ TEST(FeatureDefaultsTest, DisabledFeatures) {
 #if !BUILDFLAG(IS_ANDROID)
       &translate::kTFLiteLanguageDetectionEnabled,
 #endif
```

Comment:
are we not disabling this anymore?

Reply:
It's hard to tell in GitHub which flag you are asking about here. Typically flags that are removed from here have been deleted upstream.

Reply:
I'm referring to the removed `kWhatsNewVersion2` flag, but there is no alternative in this PR to disable the functionality

Reply:
Ah, that one is part of user education and we disable user education service. `brave://whats-new/` doesn't work.